docs/extending/admin_views.md

```diff
@@ -142,26 +142,31 @@ A 'Calendar' item will now appear in the menu.
 Sometimes you want to group custom views together in a single menu item in the sidebar. Let's create another view to display only the current calendar month:
 
 ```{code-block} python
-:emphasize-lines: 13-18
+:emphasize-lines: 15-23
 
 import calendar
-
-from django.http import HttpResponse
+from django.shortcuts import render
 from django.utils import timezone
 
 
 def index(request):
     current_year = timezone.now().year
     calendar_html = calendar.HTMLCalendar().formatyear(current_year)
 
-    return HttpResponse(calendar_html)
+    return render(request, 'wagtailcalendar/index.html', {
+        'current_year': current_year,
+        'calendar_html': calendar_html,
+    })
```

Comment:
This and below – changed because the screenshots we show above use the Wagtail template, so the tutorial was out of sync.